templates/login/partials/js/product_reg.js.twig

```diff
@@ -1,12 +1,26 @@
+
 <script src="{{ webroot|attr }}/interface/product_registration/product_registration_service.js?v={{ assetVersion|attr_url }}"></script>
 <script src="{{ webroot|attr }}/interface/product_registration/product_registration_controller.js?v={{ assetVersion|attr_url }}"></script>
-<script type="text/javascript">
-    const registrationTranslations = {{ regTranslations }};
-    const registrationConstants = {{ regConstants }};
+<script>
+    {% set regTranslations = {
+        'title': "OpenEMR Product Registration"|xla,
+        'pleaseProvideValidEmail': "Please provide a valid email address"|xla,
+        'success': "Success"|xla,
+        'registeredSuccess': "Your installation of OpenEMR has been registered"|xla,
+        'submit': "Submit"|xla,
+        'noThanks': "No Thanks"|xla,
+        'registeredEmail': "Registered email"|xla,
+        'registeredId': "Registered id"|xla,
+        'genericError': "Error. Try again later"|xla,
+        'closeTooltip': ""
+    } %}
+
+    {% set regConstants = {
+        'webroot': webroot
+    } %}
 
-    document.addEventListener('DOMContentLoaded', () => {
-        document.getElementById('authUser').focus();
-    });
+    const registrationTranslations = {{ regTranslations|json_encode|raw }};
+    const registrationConstants = {{ regConstants|json_encode|raw }};
```

Comment:
no need for the raw in above 2 lines. we do not use twig auto escaping, so no need to state that (ie. output from twig is always raw since we use our own custom escaping functions).